target_game.py

```diff
@@ -50,22 +149,55 @@ def move_target_at_random_pos(target):
         break
   
     mouse_is_pressed = mouse.getPressed()[0] == True
+    mouse_x, mouse_y = mouse.getPos()
+    
+    level_count.setText(f'Level = {level}')
+    
+    
+    #if the player does not click, the target moves and the player looses a life
+    if lives_timer.getTime() <= 0: 
+        lives -= 1
+        lives_count.setText(f'Lives = {lives}')
+        move_target_at_random_pos(target)
+        lives_timer.reset()
     
     # Check for a mouse click every 0.2s, so that we don't accept more than 1
     # press on mouse hold
     if mouse_is_pressed and mouse_click_clock.getTime() > 0.2:
         mouse_click_clock.reset()
     

```

Comment:
so I tried to run it, which didn't completely work, but I also couldn't figure out exactly whats the problem yet... I feel like I would need more psychopy knowledge for that..